### ISSUE TITLE
Milestone 3: Report feedback

### ISSUE THREAD

**Issue by ksedivyhaley** [UBC-MDS/DSCI_522_Group313#70]
- Very good responsiveness to feedback!
- Were you planning on adding the results of xgb into the final report (#55)?
- Clarity improvements:
  - First paragraph in analysis repeats some information that is now more clearly illustrated in EDA.
  - "The best set of hyperparameter for the best model: Random Forest Regressor is {‘max_depth’: 25, ‘max_features’: ‘sqrt’, ‘min_samples_split’: 2, ‘n_estimators’: 100}." : describe this in plain English.
- Minor typos

**Comment by suvarna-m** [comment]
Thanks Kate, your suggestions have been implemented here: #81 
- I added a line in Figure 2 further clarifying the xgboost test scores. We did not add full results for xgboost because it is not the model we conducted our analysis with. 
- Repetitive information in the analysis has been removed.
- Added descriptions for the hyperparameter optimizaitons
- Corrected typos (e.g. 'regresion'). 

Thanks for your feedback!
